src/js/angular/core/components/export-settings-modal/controller.js

```diff
@@ -23,7 +23,7 @@ export function ExportSettingsCtrl($scope, $uibModalInstance, LocalStorageAdapte
     $scope.defaultMode = $scope.JSONLDModes.find((mode) => mode.name === 'expanded');
     $scope.currentMode = $scope.defaultMode;
     $scope.link = null;
-
+    $scope.fileFormat = $scope.$parent.format.toLowerCase();
```

Comment:
I don't like this touching things in the parent scope. Can't you just pass this as an option to the  modal on open?

Reply:
Please see the proposed change I've made